NuRadioMC/utilities/birefringence_models/IceModel_interpolation.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import csv
+from scipy import constants
+from scipy import optimize
+from scipy import interpolate
+#import scipy.optimize
+import matplotlib.pyplot as plt
+import time
+from NuRadioMC.SignalProp import analyticraytracing as ray
+from NuRadioMC.utilities import medium
+from NuRadioMC.utilities.medium import southpole_2015
+from NuRadioReco.utilities import units
+import logging
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
add a doc string. I assume that this is never actually used and is just your script to generate the spline tables for the ice models?

Reply:
Correct, this file is only a side in case someone wants to create their own spline table. I added a doc string.